### PR TITLE
Added benchmark app for vector graphics in docs examples

### PR DESCRIPTION
**Pull request by themuflon**

### CODE REVIEW

File: docs/examples/VGBenchmark/loom.config [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "sdk_version": "dev",
```
Comment (bengarney) [inline docs/examples/VGBenchmark/loom.config]: Should be latest (you'll have to hand edit it).

File: docs/examples/VGBenchmark/src/Benchmark.ls [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+        ];
+        public static var svg:Vector.<SVG> = new Vector.<SVG>
+        [
+            SVG.fromFile("assets/loom_vector_logo_mod.svg"),
+            SVG.fromFile("assets/Hand_left.svg"),
+            SVG.fromFile("assets/nano.svg")
+        ];
+    }
+
+    public class Benchmark extends Application
```
Comment (bengarney) [inline docs/examples/VGBenchmark/src/Benchmark.ls]: Class should have a brief comment describing purpose/function of application.

**Comment by bengarney** [comment]
Nitted - also needs a README.md to match surrounding examples + screenshot. Ping me when these things are done!

**Comment by themuflon** [comment]
Optimizations!

File: loom/graphics/gfxVectorRenderer.cpp [hunk 3]
```diff
@@ -104,6 +105,7 @@ void VectorRenderer::beginFrame()
 {
     LOOM_PROFILE_SCOPE(vectorBegin);
 
+	nvgTessLevelMax(nvg, tessellationQuality);
```
Comment (bengarney) [inline loom/graphics/gfxVectorRenderer.cpp]: Should use spaces like surrounding code.

File: loom/graphics/gfxVectorRenderer.h [hunk 4]
```diff
@@ -160,6 +160,7 @@ class VectorRenderer
     static const uint8_t QUALITY_ANTIALIAS       = 1 << 0;
     static const uint8_t QUALITY_STENCIL_STROKES = 1 << 1;
     static uint8_t quality;
+	static uint8_t tessellationQuality;
```
Comment (bengarney) [inline loom/graphics/gfxVectorRenderer.h]: Tabs

File: sdk/src/loom2d/Display/Stage.ls [hunk 5]
```diff
@@ -292,6 +292,13 @@ package loom2d.display
         public native function get vectorQuality():int;
         public native function set vectorQuality(flags:int):void;
         
+        /**
+         * The recursion level of tessellation. Bigger values result in greater quality.
+         * Valid values are from 1 to 10.
```
Comment (bengarney) [inline sdk/src/loom2d/Display/Stage.ls]: Can we note 6 is now default and 5 seems to result in errors? Also, note it's maximum level of recursion. (It is isn't it?)

**Pull request merged**